lrpar/src/lib/builder.rs

```diff
@@ -274,6 +274,11 @@ pub fn parse(lexemes: &[Lexeme<{storaget}>])
         let mut cache = String::new();
         cache.push_str(" \n/* CACHE INFORMATION\n");
 
+        // Record the time that this version of lrpar was built. If the source code changes and
```

Comment:
Record the time _when_ this version of lrpar was built.

Reply:
I think either is OK, although I do marginally prefer "that". Thoughts?

Reply:
I thought when using "that" it should be: Record the time that this version of lrpar was built __at__.

It just sounded weird to me. But you're the Englishman. ;-)

Reply:
Ha, OK, let's stick with what we have then, since it's a bit shorter.

Reply:
Fair.